.github/workflows/continuous-delivery.yml

```diff
@@ -0,0 +1,146 @@
+        uses: actions/setup-java@v1
+        with:
+          java-version: 8
+
+      ## Enable Caching
+      # Reduce cache size by excluding artifacts with the project.groupId
+      - uses: actions/cache@v2
+        with:
+          path: |
+            ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-
+
+      ## Configure maven settings
+      - name: Prepare maven settings
+        env:
+          REPOSITORY_URL: "https://levigo.de/maven2/content/repositories/levigo-releases/"
+          REPOSITORY_RELEASE_USERID: ${{ secrets.PUB_NEXUS2_USERNAME }}
+          REPOSITORY_RELEASE_CREDENTIALS: ${{ secrets.PUB_NEXUS2_PASSWORD}}
+        run: |
+          mkdir -p ~/.m2
+          envsubst < ./.github/settings.xml > ~/.m2/settings.xml
+
+      ## Build with maven
+      - name: Set version
+        id: version
+        run: |
+          echo Releasing as ${{ steps.semanticversion.outputs.new_version }}
+          mvn versions:set -DnewVersion=${{ steps.semanticversion.outputs.new_version }}
+
+      - name: Perform build
+        uses: GabrielBB/xvfb-action@v1
+        with:
+          run: mvn -B verify -Dmaven.test.failure.ignore=true
+
+      ## Publish test report
+      - name: Publish Test Report for JDK 8
+        id: test-report
+        uses: scacap/action-surefire-report@v1
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          fail_on_test_failures: true
+          check_name: Test Report for JDK 8
+
+      ## Deploy
+      - name: Deploy packages
+        run: mvn -B deploy -DskipTests -Dmaven.install.skip=true
+
+      - name: Create Release
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          #don't use the GITHUB_TOKEN https://github:community/t/actions-not-triggered-on-release/17944/12:
+          GITHUB_TOKEN: ${{ secrets.REPO_PTA }}
+        with:
+          tag_name: ${{ steps.semanticversion.outputs.new_tag }}
+          release_name: ${{ steps.semanticversion.outputs.new_version }}
+
+
+      - name: Upload Release Asset - jar
+        id: upload-release-asset
+        uses: actions/upload-release-asset@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_url: ${{ steps.create_release.outputs.upload_url }} # This pulls from the CREATE RELEASE step above, referencing it's ID to get its outputs object, which include a `upload_url`. See this blog post for more info: https://jasonet.co/posts/new-features-of-github-actions/#passing-data-to-future-steps
+          asset_path: ./target/documentplatform-standard14fonts-${{ steps.semanticversion.outputs.new_version }}.jar
+          asset_name: documentplatform-standard14fonts-${{ steps.semanticversion.outputs.new_version }}.jar
+          asset_content_type: application/java-archive
+
+      - name: Upload Release Asset - sources jar
+        id: upload-release-asset
```

Comment:
so far we only uploaded one asset per release - no idea if that stunt here is really working. 

If not, plan B would be to trigger a build after the release like doing so in our web toolkit project with tomcat 9 + java 11. In the new run, build a matrix with the 3 suffixes "", "javadoc", "sources" and try to upload all 3 therefore in different runs.

Reply:
Some smaller bugs detected in the playground - project, which should be fixed now. 

@werthdavid  if you want, imho you can merge the branch now if you want.